src/corelibs/U2View/src/ov_sequence/AnnotationsTreeView.cpp

```diff
@@ -1842,6 +1844,8 @@ void AnnotationsTreeView::annotationDoubleClicked(AVAnnotationItem* item, const
 
     QList<U2Region> regionsToSelect = selectedRegions;
     const QVector<U2Region> regions = sequenceSelection->getSelectedRegions();
+    U2Region selectedRegion = selectedRegions.size() == 1 ? selectedRegions.first() : U2Region();
+    bool clearSelection = !(regions.size() == 1 && regions.first() == selectedRegion);
```

Comment:
You compute these values but never use them. Is it a bug?

Reply:
Forgot to remove, it was previous attempt to fix.